CMakeLists.txt

```diff
@@ -275,7 +281,6 @@ add_swift_library(Foundation
                   LINK_FLAGS
                     ${CoreFoundation_LIBRARIES}
                     ${ICU_UC_LIBRARY} ${ICU_I18N_LIBRARY}
-                    ${LIBXML2_LIBRARIES}
```

Comment:
This seems wrong.  The FoundationXML split hasn't occurred yet.

Reply:
This line is being removed, since `CoreFoundation_LIBRARIES` already includes the right line for libxml2.

When XML is split, it will appear in `CFXMLInterface_LIBRARIES` and that value will be used to create `FoundationXML`.